src/legacy/core_plugins/metrics/public/components/panel_config/gauge.js

```diff
@@ -38,15 +38,19 @@ import {
   EuiFormRow,
   EuiFormLabel,
   EuiSpacer,
-  EuiFieldText,
   EuiFieldNumber,
   EuiTitle,
   EuiHorizontalRule,
 } from '@elastic/eui';
 import { injectI18n, FormattedMessage } from '@kbn/i18n/react';
+import { Storage } from 'ui/storage';
+import { data } from 'plugins/data';
+import chrome from 'ui/chrome';
+const { QueryBarInput } = data.query.ui;
+const localStorage = new Storage(window.localStorage);
+const uiSettingsQueryLanguage = chrome.getUiSettingsClient().get('search:queryLanguage');
```

Comment:
nip: duplicated block. What do you think about creating a helper method for getting uiSettingsQueryLanguage. In future this approach help us to modify our code in one place  e.g.  if we need to make some refactoring and change value of search:queryLanguage

Reply:
done